sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java

```diff
@@ -673,7 +673,13 @@ private static String getConfigTemplateDownloadPath(ConfigFileSpec config) {
           .setType(rLimit.getEnum());
 
       // RLimit itself validates that both or neither of these are present.
-      if (soft.isPresent() && hard.isPresent()) {
+      if ((soft.isPresent() && hard.isPresent()) && (soft.get() != RLimitSpec.RLIMIT_INFINITY &&
+          hard.get() != RLimitSpec.RLIMIT_INFINITY))
```

Comment:
Does it make sense to add a unit test that exercises this condition as well?

Reply:
Also, should we log something in the case where only one of `soft` or `hard` are set to `RLIMIT_INFINITY`?

Reply:
We currently throw an exception in the event that only one is exclusively set: 

https://github.com/mesosphere/dcos-commons/blob/709a827ee07c0aaf01cfc54c3615196ff40aaf30/sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/RLimitSpec.java#L95

Reply:
There's this existing test case which excercises setting normal RLimits:
https://github.com/mesosphere/dcos-commons/blob/709a827ee07c0aaf01cfc54c3615196ff40aaf30/sdk/scheduler/src/test/java/com/mesosphere/sdk/specification/RLimitSpecTest.java#L8